examples/step-89/doc/results.dox

```diff
@@ -0,0 +1,92 @@
+    </td>
+    <td>
+        @image html step-89-vm-p2p.png "" width=60%
+    </td>
+  </tr>
+</table>
+
+Besides this, the results for the pressure and the velocity in $y$ direction
+are plotted along the horizontal line that spans from (0,0.69) to (1,0.69).
+Note, that the gap in the line plots (located at the position of the
+interface) is only related to the way Paraview displays the line plot. Using
+more sophisticated output generators resolves this. To focus on the important
+parts of the tutorial we do not want to add further lines of code only to
+remove the gap.
+
+<table align="center" class="doxtable">
+  <tr>
+    <td>
+        @image html step-89-p.svg "" width=60%
+    </td>
+    <td>
+        @image html step-89-uy.svg "" width=60%
+    </td>
+  </tr>
+</table>
+
+While the results of the pressure is similar, $u_y$ differs clearly. At certain
+positions we can see aliasing errors for the point-to-point interpolation.
+For different mesh configurations and/or longer run times, the aliasing effects
+of the point-to-point simulation accumulate and the simulation gets instable.
+To keep the tutorial short we have chosen one mesh that can be used for all
+examples. For a configuration that yields instable results for a wide range of
+polynomial degrees, see @cite heinz2023high.
+
+<h3>Wave propagation through in-homogeneous fluid</h3>
+
+This is just one example in which non-matching discretizations can be efficiently
+used to reduce the amount of DoFs. The example is nice, since results for a similar
+test case are shown in multiple publications. As before, we slightly adapted the
+test case to be able to use the same mesh for all simulations. The pressure field
+at $t=0.3$ is depicted below.
+
+@image html step-89-inhomogenous-p.png "" width=30%
+
+As expected, we can easily see that the wave length in the right domain is roughly
+three times times the wave length in the left domain. Hence, the wave can be
+resolved with a coarser discretization.
+
+Using the same element size in the whole domain, we can compute a reference result.
+The displayed reference result is obtained by choosing the same subdivision level
+for both sub-domains, i.e. @c subdiv_right = 11. In this particular example the
+reference result uses $92928$ DoFs, while the non-matching result uses $52608$ DoFs.
+The pressure result is plotted along the horizontal line that spans from (0,0.5) to (1,0.5).
+
+@image html step-89-inhomogenous-ref.svg "" width=30%
+
+The results, obtained with the non-matching discretization is in good agreement with
+the reference result.
+
+Note that it would be easily possible to completely, skip the influence of the non-matching
+discretization by using the in-homogeneous fluxes on all faces and by constructing a
+conforming mesh, e.g. by setting the tolerance to $1\times 10^{-6}$ during the call of
+@c GridGenerator::merge_triangulations(). However, to keep the effort low to
+recreate displayed results we refrain from doing so.
```

Comment:
THis would anyway defeat the purpose of the tutorial no? Is this paragraph necessary?

Reply:
The intention behind this paragraph is to describe how to generate a reference results that does not use non-matching implementations details (ensuring there is no problem with the non-matching implementation). 

Did you get that from my formulations?
If so, do you still have the opinion the paragraph is not necessary? :)

Reply:
I get what you mean, but i am not sure that the paragraph actually transfers this information adequately to the reader. I would still consider either removing it or improving it

Reply:
OK, I removed it :+1: